.github/workflows/maven.yml

```diff
@@ -47,7 +47,7 @@ jobs:
           cache: 'maven'
 
       - name: Build with Maven
-        run: mvn verify -e -B -V -DdistributionFileName=apache-maven
+        run: ./mvnw verify -e -B -V -DdistributionFileName=apache-maven
```

Comment:
before this step we can also add step `Set up Maven` with wrapper
and we not need add wrapper properties and scripts to code